src/main/java/org/black_ixx/bossshop/managers/config/ConfigKeyCompleter.java

```diff
@@ -28,7 +28,7 @@ private static void checkEnUsLanguage(){
         assert is != null;
         File f = new File(ClassManager.manager.getPlugin().getDataFolder(),"lang/en_us.yml");
         File f2;
-        try {f2 = File.createTempFile("en_us-lang-cache","yml");//make cache
+        try {f2 = File.createTempFile("en-US-lang-cache","yml");//make cache
```

Comment:
Can we please change this back so it is consistent with en_us.yml a couple lines earlier?